mintapi/api.py

```diff
@@ -1575,6 +1557,14 @@ def parse_arguments(args):
             ("--imap-test",),
             {"action": "store_true", "help": "Test imap login and retrieval."},
         ),
+        (
+            ("--categories",),
```

Comment:
Might be good to say that this is talking about budget categories versus all categories in transactions.

Reply:
Well it's a way to export all the categories associated with the user's Mint account.  It's not just budget categories.  That being said, I can clarify that this is a standalone option to pull your categories from Mint.